.github/workflows/release.yml

```diff
@@ -6,94 +6,76 @@ on:
       - "[0-9]+.[0-9]+.[0-9]+*"
 
 jobs:
-  analyze:
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v1
-      - uses: actions/setup-java@v1
-        with:
-          java-version: "12.x"
-      - uses: subosito/flutter-action@v1
-        with:
-          channel: "stable"
-      - name: Run CI
-        run: bash ./scripts/run_ci_tasks.sh -z
-  android:
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v1
-      - uses: actions/setup-java@v1
-        with:
-          java-version: "12.x"
-      - uses: subosito/flutter-action@v1
-        with:
-          channel: "stable"
-      - name: Run CI
-        run: bash ./scripts/run_ci_tasks.sh -a
-  ios:
+  ci:
     runs-on: macos-latest
     steps:
-      - uses: actions/checkout@v1
+      - uses: actions/checkout@v2
+      - name: Check Version
+        run: bash ./scripts/check_version.sh ${GITHUB_REF/refs\/tags\//}
       - uses: actions/setup-java@v1
         with:
           java-version: "12.x"
       - uses: subosito/flutter-action@v1
         with:
           channel: "stable"
       - name: Run CI
-        run: bash ./scripts/run_ci_tasks.sh -i
-  upload-docs:
+        run: bash ./scripts/run_ci_tasks.sh
+  release:
```

Comment:
It looks like this step is called `deploy` in the react CI, but `release` here and in Android. Would it be better to be consistent?